frontend/src/utils/fetchMajorsAndPlans.ts

```diff
@@ -69,8 +70,10 @@ export function fetchMajorsAndPlans() {
       })
         .then(res => res.json())
         .then(res => {
-          if (res.error) {
-            throw res.error;
+          if (res.statusCode >= 400) {
```

Comment:
ah last time didn't we test that statusCode only exists if you add that to the previous res? Or you checked that res.statusCode won't be null/undefined in this case?